install.sh

```diff
@@ -29,14 +29,12 @@ if [ "$(id -u)" != "0" ]; then
     [[ "$0" = "$BASH_SOURCE" ]] && exit 1 || return 1
 fi
 
-# Split path to this install script into an array.
-IFS='/' SRC_PATH=($0)
```

Comment:
I explained more about the error in the other PR #53 ... `$0` = `./install.sh` so the install directory becomes `./`, the other error is in the event that `/opt/` doesn't exist `mkdir -p /opt/` handles that